files/default/slurm/pcluster_slurm_config_generator.py

```diff
@@ -134,10 +140,55 @@ def _get_jinja_env(template_directory):
     file_loader = FileSystemLoader(template_directory)
     env = Environment(loader=file_loader, trim_blocks=True, lstrip_blocks=True)
     env.filters["sanify_instance_type"] = lambda value: re.sub(r"[^A-Za-z0-9]", "", value)
+    env.filters["gpus"] = lambda instance_type: _gpu_count(instance_type)
```

Comment:
let's rename them to: get_gpus, get_gpu_type etc, that is more clear in the jinja code